### DISCUSSION TITLE
自动选择

### DISCUSSION THREAD

**Discussion by ABC-upup** [vernesong/OpenClash#3777] [Q&A]
当我将一个策略组设置为自动选择时 就会出现ERR [Config] configuration file test failed error=proxy group[1]: ♻️ 自动选择: '美国v3-96' not found path=/etc/openclash/zhuanwaifu在线转换默认.yaml 的报错
这是怎么回事是节点的问题还是哪里设置的问题 是说这条节点不通吗 要怎么处理呢

**Comment by vernesong** [comment]
哪个插件版本
